epochStart/notifier/manualEpochStartNotifier_test.go

```diff
@@ -6,7 +6,7 @@ import (
 
 	"github.com/ElrondNetwork/elrond-go/core/check"
 	"github.com/ElrondNetwork/elrond-go/data"
-	"github.com/ElrondNetwork/elrond-go/epochStart/mock"
+	"github.com/ElrondNetwork/elrond-go/testscommon/genericmocks"
```

Comment:
genericMocks ?

Reply:
I move this mock in `testscommon/genericmocks` in order to resolve cyclic imports

Reply:
as Robert suggested, could you rename genericmocks to genericMocks?

Reply:
done.